Source/Core/VideoCommon/RenderBase.h

```diff
@@ -143,6 +144,9 @@ class Renderer
   virtual void ChangeSurface(void* new_surface_handle) {}
   bool UseVertexDepthRange() const;
 
+  // TODO: Clean this up
```

Comment:
Was this intended to be done in this PR? If not, it should probably provide some details as to what needs cleaning up about it (right now to someone who hasn't seen this PR, this is equivalent to an ambiguous "Do something")